### DISCUSSION TITLE
Supporting global diversity

### DISCUSSION THREAD

**Discussion by andrewharvey** [streetcomplete/StreetComplete#2997] [General]
Just an observation, I see all the time on the discussion threads here all sorts of claims about what quests/tags are important, what we should priorities, what one considers "spammy" and what's common and uncommon in the real world.

Myself included, we have to remember that OSM is a global project, and the world is diverse. As such we should try to avoid making decisions based on ones own personal experience and local knowledge because this is unlikely to accurately reflect the full diversity of the world.

To give some examples, Amsterdam has lots of cycleways, another city might not have a single cycleway. Another user might live in a city that only has single story buildings, another might live in Singapore where single story buildings might be rare. In one city it might be common to have to pay to use a public toilet, others it might be virtually unheard off.

So while locally you might say we can just assume this as the default, globally you can't. An unbiased and neutral StreetComplete can help to reveal some of these regional variations, but only if we design quests in a way to remain neutral.

**Comment by matkoniecz** [comment]
> we should try to avoid making decisions based on ones own personal experience and local knowledge

Note that often there is no viable alternative. And the best solution is to involve more people to get personal experience of more people and local knowledge from more places. So comments from people with different experiences are often very useful! I definitely want to run another UX test (#1470) if I will have both time and no COVID restrictions.

One can try to make decisions on statistics and general knowledge but it is far less accurate, harder to interpret and so on.

When I know that I miss some info/context/knowledge I ask other (for example I asked on talk-gb about stiles, kosher quest research was mosty asking other people ad so on)

Re https://github.com/streetcomplete/StreetComplete/issues/2944#issuecomment-868131069

> OSM doesn't have important and less important tags, it just has tags, what's important or not is subjective and depends on the contributor and data consumer.

and

> all sorts of claims about what quests/tags are important, what we should priorities, what one considers "spammy" and what's common and uncommon in the real world.

I feel comfortable saying that data from address quests is generally more useful and important that data from power pole material quest and bench backrest quest

> So while locally you might say we can just assume this as the default, globally you can't. 

I would need to check that - but I am pretty sure that SC has quests for public toilet fee  and asks about cycleways. Except countries where cycleways are so rare that this quest would be spammy - and BTW this filtering feature was added as result of personal experience of local knowledge in a different place.

**Reply by mnalis** [reply]
> Note that often there is no viable alternative

I agree. Every program by definition must have **some set of defaults**. 
Let me emphasize that: it is impossible to write a program which does not have **some defaults**.
So only thing one can work on, is choosing what that **default** will be. In SC quests situation, such **default** might be:

- **none** of the quests are selected (might be strange to new users as it seems program does nothing!)
- **all** of the quests are selected (might be overwhelming and big turn-off to new users, as they will get millions of quests many of which they wouldn't see purpose of)
- **some** of the quests are selected, which tries to strike a balance between the options above, and can again be divided:
  - **most** of the quests are selected, but few problematic ones are not (_**this is current behaviour**_)
  - **many** of the quests are selected, but some of perceived lower utility (like `material of power pole`, `roof shape`) are disabled in addition to those few problematic ones (this is how it IMHO [currently looks in #2944 it might change](https://github.com/streetcomplete/StreetComplete/issues/2944#issuecomment-867530414)) 
  - **few** of the quests are selected (just the ones the developers find most important)
  - there is some program logic where application itself decides by location, quest density and other factors which quests to display by default (my original suggestion for [#2944](https://github.com/streetcomplete/StreetComplete/issues/2944#issue-910729373))
  - user is being forced upon first startup or after some time to select quests (where there again are some defaults to start choosing from). 
  - random selection, different for each user (very bad UX IMHO, but an option that does spread the quests and is thus equally "fair" to both majority and minorities)
  - something else

Note that in all those cases **users can always enable/disable quests to their preference**, it is all about what the **default** would be. And there will always be some default, and that default will **never** be exactly to everyone's liking. One at best can hope to make it mostly sensible to majority of users.
